Scenarios/BlueGreen-Deployment-for-AKS/README.md

```diff
@@ -0,0 +1,11 @@
+# Blue Green Deployment for AKS
+This scenario is dedicated to the blue green deployment at AKS level, the main purpose is to provide a reliable and high availability solution when:
```

Comment:
Maybe revise to: This architectural pattern describes how to properly implement a Blue/Green deployment of an AKS cluster. For Blue GReen deployment at the application level, refer to [this article
](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/blue-green-spring/blue-green-spring).

Reply:
fixed